### PR TITLE
[ALLUXIO-2239] Update Alluxio security document for simple mode is enabled by default

### PR DESCRIPTION
**Pull request by sundapeng**
This PR addresses https://alluxio.atlassian.net/browse/ALLUXIO-2239

The PR is blocked by https://github.com/Alluxio/alluxio/pull/3913

### CODE REVIEW

File: docs/en/Security.md [hunk 1]
```diff
@@ -113,7 +113,9 @@ For example, the output of the shell command `ls -R` when authorization is enabl
 When user is determined, the list of groups is determined by a group mapping service, configured by
 'alluxio.security.group.mapping.class'. The default implementation is 'alluxio.security.group
 .provider.ShellBasedUnixGroupsMapping', which executes the 'groups' shell
-command to fetch the group memberships of a given user.
+command to fetch the group memberships of a given user. There is a caching mechanism for user group
+mapping, the mapping data will be cached for 60 seconds by default, the value configured by
```
Comment (ifcharming) [inline docs/en/Security.md]: nit: this value can be configured by 'alluxio.security.group.mapping.cache.timeout.ms'.

File: docs/en/Security.md [hunk 2]
```diff
@@ -136,6 +138,12 @@ The owner, group, and permissions can be changed by two ways:
 The owner can only be changed by super user.
 The group and permission can only be changed by super user and file owner.
 
+# Encryption
+
+Currently there is no service level encryption solution, user could encrypt sensitive data
```
Comment (ifcharming) [inline docs/en/Security.md]: nit: service level encryption is not supported yet.

File: docs/en/Security.md [hunk 3]
```diff
@@ -136,6 +138,12 @@ The owner, group, and permissions can be changed by two ways:
...
+at application level, or enable encryption feature at under layer file system, eg. HDFS transparent
```
Comment (ifcharming) [inline docs/en/Security.md]: nit: under file system

Comment (ifcharming) [inline docs/en/Security.md]: nit: s/eg./e.g.

File: docs/en/Security.md [hunk 4]
```diff
@@ -18,8 +18,8 @@ encryption.
 POSIX permission model is used in Alluxio to assign permissions and
 control access rights.
 
-By default Alluxio runs in non-secure mode in which no authentication and authorization are
-required.
+By default Alluxio runs in `SIMPLE` secure mode in which a simple authentication and authorization
```
Comment (ifcharming) [inline docs/en/Security.md]: can you please add more details to clarify `SIMPLE` does not actually do real authentication? meaning that server trusts whoever the client claims to be.

**Comment by sundapeng** [comment]
Thank @ifcharming for your review, I have rebased the pr, and updated according your comments.

**Comment by ifcharming** [comment]
Jenkins, test this please

**Comment by sundapeng** [comment]
Thank @ifcharming for your review.

**Pull request merged**